src/main/scala/satify/model/dpll/DecisionTree.scala

```diff
@@ -4,10 +4,10 @@ import satify.model.CNF
 import satify.model.dpll.PartialModel
 
 /** Decision is a node of DecisionTree.
-  * @param parModel The current state of the PartialModel with varName's PartialVariable constrained.
+  * @param pm The current state of the PartialModel with varName's PartialVariable constrained.
```

Comment:
maybe to clarify doc of pm

Reply:
Ok

Reply:
in "with varName's PartialVariable", i don't know what varName is, if it is not a parameter write variable name instead. maybe i'm wrong and good work for the tailrec DPLL :DD

Reply:
You're right